testsV2/mysql_test.go

```diff
@@ -58,5 +59,9 @@ func TestMysqlTcp(t *testing.T) {
 		Addr:                 "127.0.0.1:3306",
 		Net:                  "tcp",
 	}
-	proxyConnTest(t, *mysqlConnName, "mysql", cfg.FormatDSN(), mysqlPort, "")
+	ctx, cancel := context.WithTimeout(context.Background(), connTestTimeout)
+	defer cancel()
+	proxyConnTest(t,
+		ctx,
```

Comment:
Adding this context allows our tests to print a better error message when a CLI flag isn't wired up correctly. Otherwise the test just hangs indefinitely when a CLI flag is missing.

Reply:
I don't have a problem with adding a timeout, but think it can be done in proxyConnTest.